dataframe/dataframe_all_methods.go

```diff
@@ -0,0 +1,246 @@
+	"reorder_levels":    starlark.NewBuiltin("reorder_levels", methNoImpl("reorder_levels")),
+	"replace":           starlark.NewBuiltin("replace", methNoImpl("replace")),
+	"resample":          starlark.NewBuiltin("resample", methNoImpl("resample")),
+	"reset_index":       starlark.NewBuiltin("reset_index", dataframeResetIndex),
+	"rfloordiv":         starlark.NewBuiltin("rfloordiv", methNoImpl("rfloordiv")),
+	"rmod":              starlark.NewBuiltin("rmod", methNoImpl("rmod")),
+	"rmul":              starlark.NewBuiltin("rmul", methNoImpl("rmul")),
+	"rolling":           starlark.NewBuiltin("rolling", methNoImpl("rolling")),
+	"round":             starlark.NewBuiltin("round", methNoImpl("round")),
+	"rpow":              starlark.NewBuiltin("rpow", methNoImpl("rpow")),
+	"rsub":              starlark.NewBuiltin("rsub", methNoImpl("rsub")),
+	"rtruediv":          starlark.NewBuiltin("rtruediv", methNoImpl("rtruediv")),
+	"sample":            starlark.NewBuiltin("sample", methNoImpl("sample")),
+	"select_dtypes":     starlark.NewBuiltin("select_dtypes", methNoImpl("select_dtypes")),
+	"sem":               starlark.NewBuiltin("sem", methNoImpl("sem")),
+	"set_axis":          starlark.NewBuiltin("set_axis", methNoImpl("set_axis")),
+	"set_flags":         starlark.NewBuiltin("set_flags", methNoImpl("set_flags")),
+	"set_index":         starlark.NewBuiltin("set_index", methNoImpl("set_index")),
+	"shift":             starlark.NewBuiltin("shift", methNoImpl("shift")),
+	"skew":              starlark.NewBuiltin("skew", methNoImpl("skew")),
+	"slice_shift":       starlark.NewBuiltin("slice_shift", methNoImpl("slice_shift")),
+	"sort_index":        starlark.NewBuiltin("sort_index", methNoImpl("sort_index")),
+	"sort_values":       starlark.NewBuiltin("sort_values", methNoImpl("sort_values")),
+	"sparse":            starlark.NewBuiltin("sparse", methNoImpl("sparse")),
+	"squeeze":           starlark.NewBuiltin("squeeze", methNoImpl("squeeze")),
+	"stack":             starlark.NewBuiltin("stack", methNoImpl("stack")),
+	"std":               starlark.NewBuiltin("std", methNoImpl("std")),
+	"sub":               starlark.NewBuiltin("sub", methNoImpl("sub")),
+	"subtract":          starlark.NewBuiltin("subtract", methNoImpl("subtract")),
+	"sum":               starlark.NewBuiltin("sum", methNoImpl("sum")),
+	"swapaxes":          starlark.NewBuiltin("swapaxes", methNoImpl("swapaxes")),
+	"swaplevel":         starlark.NewBuiltin("swaplevel", methNoImpl("swaplevel")),
+	"tail":              starlark.NewBuiltin("tail", methNoImpl("tail")),
+	"take":              starlark.NewBuiltin("take", methNoImpl("take")),
+	"to_clipboard":      starlark.NewBuiltin("to_clipboard", methMissing("to_clipboard")),
+	"to_csv":            starlark.NewBuiltin("to_csv", methNoImpl("to_csv")),
+	"to_dict":           starlark.NewBuiltin("to_dict", methNoImpl("to_dict")),
+	"to_excel":          starlark.NewBuiltin("to_excel", methNoImpl("to_excel")),
```

Comment:
methMissing for now. I know we have an `xlsx` package, but I'd like to steer us away from excel support at this level & instead focus our energy on converting whole datasets to-and-from excel as a lossy import/export